src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -609,4 +611,20 @@ public function findPayoneOrderItemStatus(int $idSalesOrder, int $idSalesOrderIt
     public function executePartialCapture(
         PayonePartialOperationRequestTransfer $payonePartialOperationRequestTransfer
     ): CaptureResponseTransfer;
+
+    /**
+     * Specification:
+     * - Starts Payone session.
+     * - Returns client token in case session started successfully.
+     * - Returns error message if case of error.
```

Comment:
* Returns  'isSuccessful=false' and error message in case of error.

Reply:
++